### PR TITLE
PSVAMB-49001 -> Script for deployment

### PR DESCRIPTION
**Pull request by TalDubovKaltura**
@hilak I also added it to the [deployment instructions doc](https://kaltura.atlassian.net/wiki/spaces/QAC/pages/4374201330/QA.Core+SaaS+Deployment+Instructions+March+10th+2024#Specific-Fixes%2FEnhancements) -

### CODE REVIEW

**Comment by hilak** [comment]
@TalDubovKaltura  I think you should also add the script instructions to 
https://github.com/kaltura/server/blob/Tucana-20.7.0/release-notes.md

**Comment by TalDubovKaltura** [comment]
@hilak  .Done

**Review by hilak — APPROVED** [review]

File: release-notes.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Tucana-20.7.0
+### Deployment scripts ###
+    - php deployment/updates/scripts/2024_03_05_deploy_event_notification_batch_job_email_notification_failure.php
```
Comment (hilak) [inline release-notes.md:3]: @TalDubovKaltura  why did you bump the "deployment scripts" section up?

Comment (TalDubovKaltura) [reply release-notes.md:3]: @hilak This section came from the merge I did

**Pull request merged**